site/src/components/IconList.tsx

```diff
@@ -63,6 +66,28 @@ const IconList = ({icons}) => {
               key={name}
               alignItems="center"
             >
+              { contributers?.length ? (
+                <Tooltip
+                  hasArrow
+                  label="This is new or modified icon"
+                  bg={colorMode === 'light' ? theme.colors.white : theme.colors.gray[700]}
+                  color={colorMode === 'dark' ? theme.colors.white : null}
+                >
+                  <Box
+                    {
+                      ...{
```

Comment:
@ericfennis are you sure this part is correct and not giving syntax error?

Reply:
yeah this is prop destructing in react, so the object properties will be spreadout as component props